proto_test.go

```diff
@@ -2,6 +2,7 @@ package openapi2proto_test
 
 import (
 	"bytes"
+	"github.com/NYTimes/openapi2proto/protobuf"
```

Comment:
To keep with style conventions, mind grouping this import separate from the stdlib imports?

More details here: https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Thanks for the comments!
And thanks for this great project :)
I just ran `goimport`